docs/changelog.rst

```diff
@@ -9,7 +9,7 @@ ChangeLog
 *New:*
 
     - :issue:`366`: Add :class:`factory.django.Password` to generate Django :class:`~django.contrib.auth.models.User`
-      passwords.
```

Comment:
I think `passwords.` has been accidentally trimmed off the previous change log?

Reply:
Thanks, I’m not sure what happened, but I certainly did not mean to remove this word.